API/Startup.cs

```diff
@@ -22,6 +22,16 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<Iserializer, JsonSerializer>();
             services.AddSingleton<IErrorFormatter, JsonErrorFormatter>();
             services.AddControllers();
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy("AllowAll", p =>
+                {
+                    p.AllowAnyOrigin()
+                    .AllowAnyHeader()
+                    .AllowAnyMethod();
+                });
```

Comment:
Is it really wise to allow everything?

Reply:
We can later limit this to subdomains only, but it shouldn't cause any issues for the time being.